docs/relations/partner.md

```diff
@@ -1,4 +1,11 @@
-import PartnerText from './../../docs/manufacturer/approved-partner-program.mdx'
+---
+sidebar_position: 1
+sidebar_label: Approved Partner Program
+---
+
+import PartnerText from './../../docs/development/manufacturer/approved-partner-program.md'
```

Comment:
This was here before - can we have a direct path instead for better maintenance?